rpp30-boc-meltdown/biteshare
# Small fixes

## Description

- Fix for `nickname` not being displayed when user doesn't login through google/fb.
- Fix for guest bill got calculated into host account on `current session - summary` page (depends on correct `accountHolderName`/`nickname`)
- Also added a message for `current session` tab when the user hasn't created a session or joined one yet (inspired by @crystalsong2021 's 'You are currently a HOST' message lol) (I researched for disabling the icon but found someone said it's not best practice to disable a tab in your app, it's better to display a message to explain why the info is not available so I added this screen. We might want to do it for bills tab as well.)

## How to test

1. Login and go to `current session` without creating or joining a session, you should see this view: 
![Screen Shot 2022-01-03 at 3 48 53 PM](https://user-images.githubusercontent.com/43324065/147984778-eb04553c-a956-4e5d-8daf-19823b149541.png)
Clicking on 'create a session' will take you back to the explore screen, clicking on 'join' will take you to the join tab.

2. Follow what Ashe did yesterday and test with a host and a guest, the correct bill amount and status indicator should show up in the summary tab

## Notes


## Issue tracking
